src/main/java/com/practice/codingInterviewBook/deleteDuplicateInSortedArray/DeleteDuplicateInSortedArray.java

```diff
@@ -0,0 +1,47 @@
+package com.practice.codingInterviewBook.deleteDuplicateInSortedArray;
```

Comment:
Can we implement it using HashSet
It will improve the performance

Reply:
I have to check, give me some time

Reply:
okay merging it right now
let's have it in mind to implement it in HashSet.